sdk/monitor/monitor-opentelemetry-exporter/samples-dev/BasicTracerNode.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This example shows how to use 
+ * [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) 
+ * to instrument a simple Node.js application - e.g. a batch job.
+ *
+ * @summary use @opentelemetry/tracing to instrument a Node.js application
+ */
+
+import * as opentelemetry from '@opentelemetry/api';
+import { Resource } from '@opentelemetry/resources';
+import { ResourceAttributes } from '@opentelemetry/semantic-conventions';
+import { BasicTracerProvider, SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { AzureMonitorTraceExporter } from "@azure/monitor-opentelemetry-exporter";
+
+const provider = new BasicTracerProvider({
+  resource: new Resource({
+    [ResourceAttributes.SERVICE_NAME]: 'basic-service',
+  }),
+});
+
+// Configure span processor to send spans to the exporter
+const exporter = new AzureMonitorTraceExporter({
+  connectionString: process.env.APPLICATIONINSIGHTS_CONNECTION_STRING || "<your connection string>"
+});
+provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
+
+/**
+ * Initialize the OpenTelemetry APIs to use the BasicTracerProvider bindings.
+ *
+ * This registers the tracer provider with the OpenTelemetry API as the global
+ * tracer provider. This means when you call API methods like
+ * `opentelemetry.trace.getTracer`, they will use this tracer provider. If you
+ * do not register a global tracer provider, instrumentation which calls these
+ * methods will receive no-op implementations.
+ */
+provider.register();
+const tracer = opentelemetry.trace.getTracer('example-basic-tracer-node');
+
+// Create a span. A span must be closed.
+const parentSpan = tracer.startSpan('main');
+for (let i = 0; i < 10; i += 1) {
+  doWork(parentSpan);
+}
+// Be sure to end the span.
+parentSpan.end();
+
+// flush and close the connection.
+exporter.shutdown();
```

Comment:
Let's wrap this in an `async function main() { ... }` similar to how other samples do.

Then we can call it at the end of the file with

```typescript
main().catch((error) => {
  console.error("An error occurred:", error);
  process.exit(1);
});
```

This just makes it slightly nicer to the user if something throws an exception while executing the script, they won't get unhandled promise rejections.